lang/node-i2c-bus/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright (C) 2016 ?
```

Comment:
I'm not that expirienced, so whatever you do is fine with me. Also, when doing changes, maybe line 39 looks a bit too long.